contracts/DandelionOrg.sol

```diff
@@ -43,8 +40,12 @@ contract DandelionOrg is BaseTemplate {
     struct Cache {
         address dao;
         address token;
+        address finance;
         address tokenManager;
         address agentOrVault;
+        address dandelionVoting;
```

Comment:
i think that we should keep the template as simple as possible there is no need to cache apps and then pop the apps again, that will make the user pay more to deploy the org